persistent/Database/Persist/EntityDef.hs

```diff
@@ -209,12 +209,12 @@ overEntityFields
 overEntityFields f ed =
     setEntityFields (f (getEntityFieldsDatabase ed)) ed
 
--- | Gets the 'Span' of the definition of the entity.
+-- | Gets the 'SourceSpan' of the definition of the entity.
 --
 -- Note that as of this writing the span covers the entire file or quasiquote
 -- where the item is defined due to parsing limitations. This may be changed in
 -- a future release to be more accurate.
 --
--- @since 2.15.0.0
-getEntitySpan :: EntityDef -> Maybe Span
-getEntitySpan = entitySpan
```

Comment:
These are compatibility functions - let's leave them in place with the same names so migration is easier. We can deprecate in another version if we want to.

Reply:
or, I guess, the reason we provide these top-level function aliases is so we can make changes that aren't quite as painfully breaking